source/Projectile.cpp

```diff
@@ -287,35 +295,54 @@ shared_ptr<Ship> Projectile::TargetPtr() const
 
 void Projectile::CheckLock(const Ship &target)
 {
-	double base = hasLock ? 1. : .5;
+	double base = hasLock ? 1. : .15;
```

Comment:
With all the other changes made, I'm wondering if we should just leave this line alone. Missiles are already checking if they've broken lock twice as often, so reducing the change to regain a lock after that, especially by so much, seems overkill.

Reply:
In my testing it was necessary. This was an integral part of making missiles not so accurate all the time.